examples/api_access_rule/api_access_rule.tf

```diff
@@ -0,0 +1,5 @@
+resource "outscale_api_access_rule" "my_api_rule" {
+ # ca_ids      = var.ca_ids
```

Comment:
We need to provide one full example of how to setup your secure env with only one terraform apply, not one example per resource.